src/coreclr/System.Private.CoreLib/src/System/Delegate.CoreCLR.cs

```diff
@@ -435,7 +435,9 @@ private bool BindToMethodInfo(object? target, IRuntimeMethodInfo method, Runtime
         private static MulticastDelegate InternalAlloc(RuntimeType type)
         {
             Debug.Assert(type.IsAssignableTo(typeof(MulticastDelegate)));
-            return Unsafe.As<MulticastDelegate>(RuntimeTypeHandle.InternalAlloc(type));
+            MulticastDelegate result;
+            unsafe { result = Unsafe.As<MulticastDelegate>(RuntimeTypeHandle.InternalAlloc(type)); }
```

Comment:
Please ensure that all blocks `unsafe { result = Unsafe.As<MulticastDelegate>(RuntimeTypeHandle.InternalAlloc(type)); }`

are formatted

```
unsafe
{
    result = Unsafe.As<MulticastDelegate>(RuntimeTypeHandle.InternalAlloc(type));
}
```